src/main/drivers/resource.c

```diff
@@ -113,6 +113,6 @@ const char * const ownerNames[OWNER_TOTAL_COUNT] = {
     "RX_SPI_EXPRESSLRS_BUSY",
     "SOFTSERIAL_TX",
     "SOFTSERIAL_RX",
-    [OWNER_LPUART_TX] = "LPUART_TX",
```

Comment:
See https://github.com/betaflight/betaflight/issues/13567#issuecomment-2070384714

Right now I feel it's important to be consistant, and use the same style as all the other lines in `ownerNames`, since the above syntax seems not to work as intended.